File: WMS/app/src/main/java/com/example/wms/employee.java
package com.example.wms;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;



public class employee extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.employee);
        CardView cardschedule=findViewById(R.id.card_workschedule);
        CardView carddutyhours=findViewById(R.id.card_dutyhours);
        CardView cardworkhistory=findViewById(R.id.card_workhistory);
        CardView cardassignedwork=findViewById(R.id.card_assignedwork);
        CardView cardapplyleave=findViewById(R.id.card_applyleave);
        CardView cardtransfer=findViewById(R.id.card_transfer);

        cardschedule.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent scheduleintent=new Intent(employee.this, workschedule.class);
                startActivity(scheduleintent);
            }
        });

        carddutyhours.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent dutyintent=new Intent(employee.this, workschedule.class);
                startActivity(dutyintent);
            }
        });

        cardworkhistory.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent historyintent=new Intent(employee.this, workhistory.class);
                startActivity(historyintent);
            }
        });
        cardassignedwork.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent assignedintent=new Intent(employee.this, assignedwork.class);
                startActivity(assignedintent);
            }
        });

        cardapplyleave.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent leaveintent=new Intent(employee.this, leave.class);
                startActivity(leaveintent);
            }
        });


        cardtransfer.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent transferintent=new Intent(employee.this, transfer.class);
                startActivity(transferintent);
            }
        });








    }
}

File: app/src/main/java/com/example/wms/ui/employee/employee.kt
package com.example.wms.ui.employee

import android.content.Intent
import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.cardview.widget.CardView
import com.example.wms.R
import com.example.wms.assignedwork
import com.example.wms.leave
import com.example.wms.transfer
import com.example.wms.workhistory
import com.example.wms.workschedule

class employee : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        this.enableEdgeToEdge()
        setContentView(R.layout.employee)
        val cardschedule = findViewById<CardView>(R.id.card_workschedule)
        val carddutyhours = findViewById<CardView>(R.id.card_dutyhours)
        val cardworkhistory = findViewById<CardView>(R.id.card_workhistory)
        val cardassignedwork = findViewById<CardView>(R.id.card_assignedwork)
        val cardapplyleave = findViewById<CardView>(R.id.card_applyleave)
        val cardtransfer = findViewById<CardView>(R.id.card_transfer)

        cardschedule.setOnClickListener {
            val scheduleintent = Intent(
                this@employee,
                workschedule::class.java
            )
            startActivity(scheduleintent)
        }

        carddutyhours.setOnClickListener {
            val dutyintent = Intent(
                this@employee,
                workschedule::class.java
            )
            startActivity(dutyintent)
        }

        cardworkhistory.setOnClickListener {
            val historyintent = Intent(
                this@employee,
                workhistory::class.java
            )
            startActivity(historyintent)
        }
        cardassignedwork.setOnClickListener {
            val assignedintent = Intent(
                this@employee,
                assignedwork::class.java
            )
            startActivity(assignedintent)
        }

        cardapplyleave.setOnClickListener {
            val leaveintent = Intent(
                this@employee,
                leave::class.java
            )
            startActivity(leaveintent)
        }


        cardtransfer.setOnClickListener {
            val transferintent = Intent(
                this@employee,
                transfer::class.java
            )
            startActivity(transferintent)
        }
    }
}